synapse/replication/tcp/redis.py

```diff
@@ -206,34 +255,37 @@ def __init__(
         self, hs: "HomeServer", outbound_redis_connection: txredisapi.RedisProtocol
     ):
 
-        super().__init__()
-
-        # This sets the password on the RedisFactory base class (as
-        # SubscriberFactory constructor doesn't pass it through).
-        self.password = hs.config.redis.redis_password
+        super().__init__(
+            hs,
+            "subscriber",
+            None,
+            1,
```

Comment:
It would probably be clearer to use kwargs for these. (`None` and `1` seem to be dbid and poolsize, but I had to fliip back and forth between classes to see that.)